### PR TITLE
Make the api-url required and add branded checkout server configurati…

### PR DESCRIPTION
**Pull request by OzzieOrca**
…on docs

You can click the little rich diff button which is cool
screen shot 2018-09-25 at 4 02 24 pm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +133,36 @@ The `<branded-checkout>` element is where the branded checkout Angular app will
   - `$event.$window` - Provides access to the browser's global `window` object. This allows you to call a custom callback function like `onOrderCompleted` in the example.
   - `$event.purchase` - contains the order's details that are loaded for the thank you page
 
+#### Server-side configuration for a new branded checkout domain
+1. Figure out what domain you will be hosting the branded checkout form on. For example, `myministry.org`
+2. Make sure HTTPS is enabled on that domain
+3. You will need to setup a subdomain for the give.cru.org API. We've experienced CORS issues and cross-domain cookie issues trying to hit the give.cru.org API directly from a custom domain. Create a CNAME record for `brandedcheckout.myministry.org` (the subdomain could be different but using that suggested subdomain makes it consistent with other sites) and point it at `give.cru.org`.
```
Comment (mattdrees) [inline README.md]: I think CORS is not an issue; cross-domain cookies are the only issue. FWIW.

**Comment by mattdrees** [comment]
thanks scotty; looks good

File: README.md (hunk 1 above)
Comment (OzzieOrca) [reply README.md]: I can remove that line. It is an issue if you don't configure it at https://maintenance-production-alb-1117343512.us-east-1.elb.amazonaws.com. Lol. Thanks for sharing that again.

File: README.md [hunk 2]
```diff
@@ -131,6 +133,36 @@ The `<branded-checkout>` element is where the branded checkout Angular app will
...
+4. In order to accept credit cards on your own domain, you will need to setup a TSYS merchant account. Contact the Cru's Financial Services Group ([jesse.kimbell@cru.org](mailto:jesse.kimbell@cru.org)) if you don't already have one. You will need a TSYS device id (a numeric id around 14 digits) to complete the next step.
```
Comment (wrandall22) [inline README.md]: @hlbraddock is it still Jesse Kimbell?

Comment (hlbraddock) [reply README.md]: Chad Vaughan.

**Pull request merged**